drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -222,6 +242,7 @@
     display: none;
 }
 .xblock--drag-and-drop--editor .items-form .row.advanced-link {
+    cursor: pointer;
```

Comment:
Yes, that's right. I removed `href="#"` and added this CSS rule. I believe this is considered to be a better approach than `href="#"`, which has always felt like a hack to me.